CMakeLists.txt

```
@@ -1,14 +1,38 @@
-cmake_minimum_required(VERSION 2.8.3)
-
-project(modelo_carrinho)
-
-find_package(catkin REQUIRED)
-
-catkin_package()
-
-find_package(roslaunch)
-
-foreach(dir config launch meshes urdf)
-	install(DIRECTORY ${dir}/
-		DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
-endforeach(dir)
+cmake_minimum_required(VERSION 3.0.0)
```

O ROS noetic recomenda a versão 3.0.2, item 2 do [guia de migração](http://wiki.ros.org/noetic/Migration)

```suggestion
cmake_minimum_required(VERSION 3.0.2)
```